src/audio/PlaylistTrack.js

```diff
@@ -101,6 +117,9 @@ FORGE.PlaylistTrack.prototype._parseConfig = function(config)
     this._author = new FORGE.LocaleString(this._viewer, config.author);
     this._url = config.url;
     this._loop = config.loop || false;
+
+    this._autoPause = (typeof config.autoPause !== "undefined") ? !!config.autoPause : null;
+    this._autoResume = (typeof config.autoResume !== "undefined") ? !!config.autoResume : null;
```

Comment:
Same as before :)
this._autoPause = (typeof config.autoPause == "boolean") ? config.autoPause : defaultValue;